### DISCUSSION TITLE
Localization?

### DISCUSSION THREAD

**Discussion by alvarlagerlof** [alvarlagerlof/neurodiversity#295] [Ideas]
In the long-term, we might want to translate pages to different languages and show the right one based on device language. I can do translations to Swedish. But more languages would be better.

**Comment by pedroapfilho** [comment]
I can add translations for portuguese.

The first step would be to create the initial setup with `react-intl` or some other package to handle translations. Would you like someone else to create this setup, or would you like to create it yourself? We also need to separate the content into different language folders.
